torchvision/io/video.py

```diff
@@ -179,7 +192,9 @@ def _align_audio_frames(aframes, audio_frames, ref_start, ref_end):
     return aframes[:, s_idx:e_idx]
 
 
-def read_video(filename, start_pts=0, end_pts=None, pts_unit="pts"):
+def read_video(
+    filename: str, start_pts: int = 0, end_pts: Optional[float] = None, pts_unit: str = "pts"
```

Comment:
just make `end_pts` be an optional of float here for now I think

Reply:
Do you mean `start_pts`? `end_pts` is already an `Optional[float]`.

Reply:
Sorry, I meant `Optional[int]`